Gemfile

```diff
@@ -15,7 +15,7 @@ if next?
   gem "rails", github: "rails/rails", branch: "main"
   gem "devise", github: "heartcombo/devise", branch: "main"
 else
-  gem "rails", "~> 6.1.4"
+  gem "rails", "~> 7.0.2.4"
```

Comment:
maybe use `~> 7.0.2` here? the current version will only allow `7.0.2.5/6/7/8..` but not `7.0.3` for example, and we usually limit the minor version but not the patch version